dist/ami/files/99_user_alias.cfg

```diff
@@ -0,0 +1,4 @@
+runcmd:
+  - /sbin/userdel -r ec2-user
+  - /sbin/groupdel ec2-user
+  - /sbin/useradd -o -u 1001 -g scyllaadm -d /home/scyllaadm centos
```

Comment:
This doesn't create a scyllaadm user, it creates a centos user, no?

I see the -o option, but don't we need a useradd before?

Reply:
We create scyllaadm user on cloud-init, instead of ec2-user.
Then cloud-init runs this script after user creation.
https://github.com/scylladb/scylla/commit/efd0fc9f769e9e6020031bbd9b6bc399e5219800#diff-2fe20e870f705ad288eafddbd5031182R90